index.html

```diff
@@ -131,18 +131,19 @@ <h3>Third Slide</h3>
     <!-- Page Content -->
     <div class="container">
   
-      <h1 class="my-4">Welcome to Modern Business</h1>
+      <h1 class="my-4">Welcome to Poetry</h1>
   
       <!-- Marketing Icons Section -->
       <div class="row">
         <div class="col-lg-4 mb-4">
           <div class="card h-100">
-            <h4 class="card-header">Card Title</h4>
```

Comment:
I see the conflict you mentioned here, I've adjusted my Violet jacobs branch to avoid the issue. Good work!
`            <h4 class="card-header">Card Title</h4>
            <h4 class="card-header">Edgar Allan Poe</h4>
            <a href="#"><img class="card-img-top" src="images/poe.jpeg" alt="A watercolor of a raven"></a>
            <div class="card-body">
              <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente esse necessitatibus neque.</p>
              <p class="card-text">Put in a intro to the bio.</p>
            </div>
            <div class="card-footer">
              <a href="#" class="btn btn-primary">Learn More</a>
              <a href="https://en.wikipedia.org/wiki/Edgar_Allan_Poe" class="btn btn-primary">Learn more about Poe.</a>`